bookkeeper-dist/src/main/resources/LICENSE-all.bin.txt

```diff
@@ -202,507 +202,265 @@
-   PROHIBITS SUCH LIMITATION. SOME JURISDICTIONS DO NOT ALLOW THE
-   EXCLUSION OR LIMITATION OF INCIDENTAL OR CONSEQUENTIAL DAMAGES, SO
-   THIS EXCLUSION AND LIMITATION MAY NOT APPLY TO YOU.
-
-8. U.S. GOVERNMENT END USERS.
-
-   The Covered Software is a .commercial item,. as that term is
-   defined in 48 C.F.R. 2.101 (Oct. 1995), consisting of .commercial
-   computer software. (as that term is defined at 48 C.F.R. ?
-   252.227-7014(a)(1)) and .commercial computer software
-   documentation. as such terms are used in 48 C.F.R. 12.212
-   (Sept. 1995). Consistent with 48 C.F.R. 12.212 and 48
-   C.F.R. 227.7202-1 through 227.7202-4 (June 1995), all
-   U.S. Government End Users acquire Covered Software with only those
-   rights set forth herein. This U.S. Government Rights clause is in
-   lieu of, and supersedes, any other FAR, DFAR, or other clause or
-   provision that addresses Government rights in computer software
-   under this License.
-
-9. MISCELLANEOUS.
-
-   This License represents the complete agreement concerning subject
-   matter hereof. If any provision of this License is held to be
-   unenforceable, such provision shall be reformed only to the extent
-   necessary to make it enforceable. This License shall be governed by
-   the law of the jurisdiction specified in a notice contained within
-   the Original Software (except to the extent applicable law, if any,
-   provides otherwise), excluding such jurisdiction.s conflict-of-law
-   provisions. Any litigation relating to this License shall be
-   subject to the jurisdiction of the courts located in the
-   jurisdiction and venue specified in a notice contained within the
-   Original Software, with the losing party responsible for costs,
-   including, without limitation, court costs and reasonable
-   attorneys. fees and expenses. The application of the United Nations
-   Convention on Contracts for the International Sale of Goods is
-   expressly excluded. Any law or regulation which provides that the
-   language of a contract shall be construed against the drafter shall
-   not apply to this License. You agree that You alone are responsible
-   for compliance with the United States export administration
-   regulations (and the export control laws and regulation of any
-   other countries) when You use, distribute or otherwise make
-   available any Covered Software.
-
-10. RESPONSIBILITY FOR CLAIMS.
-
-   As between Initial Developer and the Contributors, each party is
-   responsible for claims and damages arising, directly or indirectly,
-   out of its utilization of rights under this License and You agree
-   to work with Initial Developer and Contributors to distribute such
-   responsibility on an equitable basis. Nothing herein is intended or
-   shall be deemed to constitute any admission of liability.
+lib/io.netty-netty-all-4.1.12.Final.jar contains a modified portion of and optionally depends on 'Protocol Buffers', Google's data
```

Comment:
This is optionally dependency. why do we need this?

Reply:
"contains a modified portion of" implies that they are pulling code directly in.

Reply:
I think it follows into the above comment. We need to address the comment above if you feel strong on using this approach here.

This also a problem I can see here. We are unclear about what our dependencies do and what they changed. We should just include our dependencies' license and notice, we should not import their licenses, unless that is the dependency of dependency is explicitly included in our bundle.

Last, protobuf is already in the notice/license in our bundle. I am not sure why we still need a separate one.

Reply:
Again, this is code copied in, since it is modified. As it is copied in, we are distributing it, we should distribute the license for it.